### ISSUE TITLE
Set up a recurring reminder

### ISSUE THREAD

**Issue by jamesmenera** [cg-cnu/vscode-remind-me#7]
This is a feature request. But I would like to set up something that bugs me every hour or something to remind me to do something. Maybe it is to stand up, stretch, commit code, etc.

**Comment by mphelps1978** [comment]
I want to upvote this a million times 

something like "Commit your code every 30 minutes" and make it a popup alert instead of a status notification

**Comment by singhmj** [comment]
Yeah, this feature would be really helpful. I am thinking to use it like "Remind me to take a short break every hour from now".
